enterprise/jakarta.transformer/manifest.mf

```diff
@@ -0,0 +1,8 @@
+Manifest-Version: 1.0
+OpenIDE-Module: org.netbeans.modules.jakarta.transformer/0
+OpenIDE-Module-Layer: org/netbeans/modules/fish/payara/jakarta/transformer/layer.xml
+OpenIDE-Module-Localizing-Bundle: org/netbeans/modules/fish/payara/jakarta/transformer/Bundle.properties
+AutoUpdate-Show-In-Client: true
+AutoUpdate-Essential-Module: true
+OpenIDE-Module-Specification-Version: 2.15
```

Comment:
@neilcsmith-net @matthiasblaesing do you know what `AutoUpdate-Essential-Module` does? I looked a bit at the module system code and I still don't know :)

Reply:
Only reading code I see:
- an essential module can't be deleted
- an essential module can't be disabled

Reply:
@matthiasblaesing thanks. So lets see if we can remove that flag.